tests/unit/dynamic_dependency_test.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
In fact, this is already fixed on master, just a missing cherry-pick, which I fixed here: https://github.com/cfengine/core/commit/0000c548d3c70204c68d897fa99f6468272396d0

Thanks for bringing it up though!